extensions/sql-database-projects/src/controllers/projectController.ts

```diff
@@ -185,29 +185,29 @@ export class ProjectsController {
 	}
 
 	/**
-	 * Builds and deploys a project
+	 * Builds and publishes a project
 	 * @param treeNode a treeItem in a project's hierarchy, to be used to obtain a Project
 	 */
-	public async deployProject(treeNode: BaseProjectTreeItem): Promise<DeployDatabaseDialog>;
+	public async publishProject(treeNode: BaseProjectTreeItem): Promise<PublishDatabaseDialog>;
 	/**
-	 * Builds and deploys a project
-	 * @param project Project to be built and deployed
+	 * Builds and publishes a project
+	 * @param project Project to be built and published
 	 */
-	public async deployProject(project: Project): Promise<DeployDatabaseDialog>;
-	public async deployProject(context: Project | BaseProjectTreeItem): Promise<DeployDatabaseDialog> {
+	public async publishProject(project: Project): Promise<PublishDatabaseDialog>;
+	public async publishProject(context: Project | BaseProjectTreeItem): Promise<PublishDatabaseDialog> {
 		const project: Project = this.getProjectFromContext(context);
-		let deployDatabaseDialog = this.getDeployDialog(project);
+		let publishDatabaseDialog = this.getPublishDialog(project);
 
-		deployDatabaseDialog.deploy = async (proj, prof) => await this.executionCallback(proj, prof);
-		deployDatabaseDialog.generateScript = async (proj, prof) => await this.executionCallback(proj, prof);
-		deployDatabaseDialog.readPublishProfile = async (profileUri) => await this.readPublishProfile(profileUri);
+		publishDatabaseDialog.publish = async (proj, prof) => await this.executionCallback(proj, prof);
+		publishDatabaseDialog.generateScript = async (proj, prof) => await this.executionCallback(proj, prof);
+		publishDatabaseDialog.readPublishProfile = async (profileUri) => await this.readPublishProfile(profileUri);
 
-		deployDatabaseDialog.openDialog();
+		publishDatabaseDialog.openDialog();
 
-		return deployDatabaseDialog;
+		return publishDatabaseDialog;
 	}
 
-	public async executionCallback(project: Project, profile: IDeploymentProfile | IGenerateScriptProfile): Promise<mssql.DacFxResult | undefined> {
+	public async executionCallback(project: Project, profile: IPublishProfile | IGenerateScriptProfile): Promise<mssql.DacFxResult | undefined> {
```

Comment:
it's a little confusing now that there's IPublishProfile and there's also the publish profile file that can be loaded. Any ideas on if one should be renamed to something else?

Reply:
I think these two interfaces can be renamed to something not-Profile.  Maybe settings or configuration?

Reply:
Changed them to settings